mani_skill/agents/robots/xarm6/xarm6_xarmgripper.py

```diff
@@ -0,0 +1,316 @@
+            self.arm_friction,
+            normalize_action=False,
+        )
+        arm_pd_joint_delta_pos_vel = PDJointPosVelControllerConfig(
+            self.arm_joint_names,
+            -0.1,
+            0.1,
+            self.arm_stiffness,
+            self.arm_damping,
+            self.arm_force_limit,
+            friction=self.arm_friction,
+            use_delta=True,
+        )
+
+        # -------------------------------------------------------------------------- #
+        # Gripper
+        # -------------------------------------------------------------------------- #
+        # NOTE(jigu): IssacGym uses large P and D but with force limit
+        # However, tune a good force limit to have a good mimic behavior
+        gripper_pd_joint_pos = PDJointPosMimicControllerConfig(
+            self.gripper_joint_names,
+            lower=-0.01,  # a trick to have force when the object is thin
+            upper=0.04,
+            stiffness=self.gripper_stiffness,
+            damping=self.gripper_damping,
+            force_limit=self.gripper_force_limit,
+            friction=self.gripper_friction,
+        )
+        # gripper_pd_joint_pos = PassiveControllerConfig(
+        #     self.gripper_joint_names,
+        #     damping=self.gripper_damping,
+        #     force_limit=self.gripper_force_limit,
+        # )
+
+        controller_configs = dict(
+            pd_joint_delta_pos=dict(
+                arm=arm_pd_joint_delta_pos, gripper=gripper_pd_joint_pos
+            ),
+            pd_joint_pos=dict(arm=arm_pd_joint_pos, gripper=gripper_pd_joint_pos),
+            pd_ee_delta_pos=dict(arm=arm_pd_ee_delta_pos, gripper=gripper_pd_joint_pos),
+            pd_ee_delta_pose=dict(
+                arm=arm_pd_ee_delta_pose, gripper=gripper_pd_joint_pos
+            ),
+            pd_ee_pose=dict(arm=arm_pd_ee_pose, gripper=gripper_pd_joint_pos),
+            # TODO(jigu): how to add boundaries for the following controllers
+            pd_joint_target_delta_pos=dict(
+                arm=arm_pd_joint_target_delta_pos, gripper=gripper_pd_joint_pos
+            ),
+            pd_ee_target_delta_pos=dict(
+                arm=arm_pd_ee_target_delta_pos, gripper=gripper_pd_joint_pos
+            ),
+            pd_ee_target_delta_pose=dict(
+                arm=arm_pd_ee_target_delta_pose, gripper=gripper_pd_joint_pos
+            ),
+            # Caution to use the following controllers
+            pd_joint_vel=dict(arm=arm_pd_joint_vel, gripper=gripper_pd_joint_pos),
+            pd_joint_pos_vel=dict(
+                arm=arm_pd_joint_pos_vel, gripper=gripper_pd_joint_pos
+            ),
+            pd_joint_delta_pos_vel=dict(
+                arm=arm_pd_joint_delta_pos_vel, gripper=gripper_pd_joint_pos
+            ),
+        )
+
+        # Make a deepcopy in case users modify any config
+        return deepcopy_dict(controller_configs)
+
+
+    @property
+    def _sensor_configs(self):
```

Comment:
normally we like to have a with and without mounted camera variant of the robot.

A simple solution is to create a new class below this one called xarm6_xarmgripper_wristcam or something and have it inherit the original class but override the sensor configs.

Reply:
Fixed! Separated xarm6 variants with and without wristcam.